### PR TITLE
gcp - iam init documentation

### PR DESCRIPTION
**Pull request by mediapills**
Init documentation for IAM resource

### CODE REVIEW

File: docs/source/gcp/examples/iam-role-create.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+IAM - Watch for Creation of New Roles
+============================================================
```
Comment (kapilt) [inline docs/source/gcp/examples/iam-role-create.rst]: having 200 examples of the same thing isn't particular useful. let's stop with the resource create, notify examples we have enough.

Comment (mediapills) [reply docs/source/gcp/examples/iam-role-create.rst]: okay, I will close this PR and after actions, implementation will provide a better example

Comment (mediapills) [reply docs/source/gcp/examples/iam-role-create.rst]: okay, I will close this PR and after actions, implementation will provide a better example

**Comment by mediapills** [comment]
waiting for IAM actions implementation

**Pull request closed without merging**